packages/opencensus-core/src/metrics/metric-registry.ts

```diff
@@ -228,6 +218,66 @@ export class MetricRegistry {
     return doubleCumulative;
   }
 
+  /**
+   * Builds a new derived Int64 Cumulative to be added to the registry.
+   *
+   * @param name The name of the metric.
+   * @param options The options for the metric.
+   * @returns A Int64 DerivedCumulative metric.
+   */
+  addDerivedInt64Cumulative(name: string, options?: MetricOptions):
+      DerivedCumulative {
+    const description =
```

Comment:
Seems like some of this code is duplicated with other metric adding functions. That isn't necessarily bad but makes me wonder if there would be a more templatized approach to the different metric types. (On the other hand, [Duplication is better than the wrong abstraction](https://www.sandimetz.com/blog/2016/1/20/the-wrong-abstraction) so I think this is fine)

Reply:
Ok, thanks.